examples/40_advanced/example_resampling.py

```diff
@@ -89,26 +90,39 @@
 # =============================
 # It is also possible to use
 # `scikit-learn's splitter classes <https://scikit-learn.org/stable/modules/classes.html#splitter
-# -classes>`_ to further customize the outputs.
+# -classes>`_ to further customize the outputs. In case one needs to have 100% control over the
+# splitting, it is possible to use
+# `scikit-learn's PredefinedSplit <https://scikit-learn.org/stable/modules/generated/
+# sklearn.model_selection.PredefinedSplit.html>`_.
 
+############################################################################
+# Exemplary use of a predefined split. We split the training data by the first
+# feature. In practice, one would use a splitting according to the use case
+# at hand.
 
-resampling_strategy = sklearn.model_selection.RepeatedStratifiedKFold
-resampling_strategy_arguments = {'folds': 3}
+resampling_strategy = sklearn.model_selection.PredefinedSplit
+resampling_strategy_arguments = {'test_fold': np.where(X_train[:, 0] < np.mean(X_train[:, 0]) )[0]}
 
 automl = autosklearn.classification.AutoSklearnClassifier(
     time_left_for_this_task=120,
     per_run_time_limit=30,
     tmp_folder='/tmp/autosklearn_resampling_example_tmp',
     output_folder='/tmp/autosklearn_resampling_example_out',
     disable_evaluator_output=False,
-    resampling_strategy='cv',
-    resampling_strategy_arguments={'folds': 5},
+    resampling_strategy=resampling_strategy,
+    resampling_strategy_arguments=resampling_strategy_arguments,
 )
 automl.fit(X_train, y_train, dataset_name='breast_cancer')
 
 ############################################################################
-# Get the Score of the final ensemble
-# ===================================
+# For unknown resampling strategies it is necessary to perform a refit
```

Comment:
Hi Matthias, what do you mean by unknown resampling strategies?
Do you mean, custom strategies in general, like the ones not listed in the API?

Reply:
Yes, updated the example.